docs/conversation.md

```diff
@@ -104,6 +104,7 @@
 | `participants`          | array   | v1    | v2      | **Removed**                                                                                                                                                                                                                                                                                                                                                                                       |
 | `guestList`             | string  | v1    | v2      | **Removed**                                                                                                                                                                                                                                                                                                                                                                                       |
 | `avatarVersion`         | string  | v4    |         | Version of conversation avatar used to easier expiration of the avatar in case a moderator updates it, since the avatar endpoint should be cached for 24 hours.                                                                                                                                                                                                                                   |
```

Comment:
```suggestion
| `avatarVersion`         | string  | v4    |         | Version of conversation avatar used to easier expiration of the avatar in case a moderator updates it, since the avatar endpoint should be cached for 24 hours. (only available with `avatar` capability)                                                                                                                                                                                                                                      |
```